_includes/navbar.html

```diff
@@ -14,6 +14,15 @@
                 }
             })
         });
+        $(function() {
+        var url = window.location.href;
+        $("nav a").each(function() {
+            console.log(url, this.href);
```

Comment:
You should remove this debugging statement?

Reply:
Ya sure, Thanks for pointing it out.
Removing in the next commit